app/src/test/java/org/simple/clinic/scanid/ScanSimpleIdEffectHandlerTest.kt

```diff
@@ -203,4 +208,40 @@ class ScanSimpleIdEffectHandlerTest {
     verify(uiActions).openPatientSearch(identifier, null, patientPrefillInfo)
     verifyNoMoreInteractions(uiActions)
   }
+
+  @Test
+  fun `when online patient lookup with identifier is received, then load the online lookup api and patient not found`() {
```

Comment:
We can rename the test a bit

`when online patient lookup with identifier effect is received, then lookup patients online with the patient not found error`